modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/SimplePlayer.kt

```diff
@@ -230,7 +230,13 @@ class SimplePlayer(val settings: Settings, val statsManager: StatsManager, val c
             .setUserAgent("Pocket Casts")
             .setAllowCrossProtocolRedirects(true)
         val dataSourceFactory = DefaultDataSource.Factory(context, httpDataSourceFactory)
-        val extractorsFactory = DefaultExtractorsFactory().setMp3ExtractorFlags(Mp3Extractor.FLAG_ENABLE_CONSTANT_BITRATE_SEEKING)
+        val extractorsFactory = DefaultExtractorsFactory().setMp3ExtractorFlags(
+            if (settings.seekToAccuracy()) {
+                Mp3Extractor.FLAG_ENABLE_INDEX_SEEKING
+            } else {
+                Mp3Extractor.FLAG_ENABLE_CONSTANT_BITRATE_SEEKING
+            }
```

Comment:
Would it be possible to only do this for variable bit rate files? I'm guessing that by the time we know the file is variable bit rate it's probably too late, but figured I'd ask in case I'm mistaken.